acceptance-tests/dsl/src/main/java/org/hyperledger/besu/tests/acceptance/dsl/blockchain/Amount.java

```diff
@@ -17,8 +17,6 @@
 import static org.web3j.utils.Convert.Unit.ETHER;
 import static org.web3j.utils.Convert.Unit.WEI;
 
-import org.hyperledger.besu.ethereum.core.Wei;
```

Comment:
We should not replace Wei with a type that is shared across multiple units.  We need this as `Wei` for type and unit safety.